prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -90,6 +93,15 @@ class FHIRConverter(
                 // conduct FHIR Transform
                 transformer?.transform(bundle)
 
+                // 'stamp' observations with their condition code
+                bundle.entry.map { it.resource }.filterIsInstance<Observation>().forEach {
+                    it.addMappedCondition(metadata).run {
+                        actionLogger.getItemLogger(bundleIndex + 1, it.id)
+                            .setReportId(message.reportId)
+                            .log(this, ActionLogLevel.warning)
```

Comment:
Nit: can this be switched to just using `warning` and then making `log` private again?